examples/dlbenchmark/pytorch/rossmann-store-sales-dlbenchmark-hpo.ipynb

```diff
@@ -0,0 +1,769 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## HPO for NVTabular DL"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "NVTabular data loader for PyTorch showed 10x-20x speedup in comparison to the native PyTorch data loader. This notebook addresses the question, **does the NVTabular data loader convergeces to the same performance as the native PyTorch data loader and in which umber of epochs?**<br><br>\n",
+    "\n",
+    "In this notebook, we analyse the convergence performance on the Rossmann Sales dataset.<br><br>\n",
+    "\n",
+    "First, we train the native PyTorch data loader five times for 750 epochs to establish a baseline and get the variance\n",
+    "Second, we train the NVtabular data loader five times for 750 epochs with different configurations:\n",
+    "- part_size: ['10MB', '50MB', '100MB', '500MB', '1GB'] \n",
+    "- shuffle: [False, True]\n",
+    "- dl_part_per_chunk: [1, 2, 4, 8, 16]\n",
+    "<br><br><br>\n",
+    "We ran in total 255 experiments, 5 for native PyTorch data loader and 250 for NVTabular data loader."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will show in this notebook:<br>\n",
+    "* If both data loaders are trained to **full convergence (750 epochs)**, NVTabular data loader for PyTorch achieves **slightly lower valid RMSPE** than native PyTorch data loader but **requires only 1/8 of the computation time**.\n",
+    "* If both data loaders have the **same compuational budget (time)**, then NVTabular data loader for PyTorch achieves **~24% lower valid RMSPE**.\n",
+    "* The distributions shows that NVTabular data loader convergence similar to native PyToch data loader\n",
+    "* NVTabular data loader achieves **same valid RMSPE amlost independent of the hyperparameters*. **80% of the experiments achieve lower valid RMSPE**\n",
+    "* Analyzing the different configuration for NVTabular data loader shows no trend, **enabling the NVTabular data loader ready-to-use without hyperparameter tuning**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pickle\n",
+    "import glob\n",
+    "\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "255"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "files = glob.glob('./output/*.pickle')\n",
+    "len(files)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "list_df = []\n",
+    "\n",
+    "for file in files:\n",
```

Comment:
Why are you loading in the dataframe this way? What is the benefit of using a pickle here... Instead of a parquet file? Is the pickle object carrying other non dataframe-type data?